keyboards/bastardkb/dilemma/4x6_4/config.h

```diff
@@ -42,3 +42,6 @@
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+// Reduce soft serial speed: Work around rp2040 issues
+#define SELECT_SOFT_SERIAL_SPEED 4
```

Comment:
This is probably fixed now, as there were some fixes for the split transport last cycle.  That said, `SERIAL_USART_SPEED` would be better here, IMO
```suggestion
#define SERIAL_USART_SPEED 38400
```

Reply:
I'll test whether this is still needed and remove or change as appropriate.

Reply:
Confirmed, it's no longer required: I could no longer reproduce the immediate troubles we used to have. Removed.